src/renderer/components/+preferences/preferences.tsx

```diff
@@ -259,7 +259,7 @@ export class Preferences extends React.Component {
           <section id="telemetry">
             <h2 data-testid="telemetry-header">Telemetry</h2>
             {telemetryExtensions.map(this.renderExtension)}
-            {sentryDsnIsValid ? (
+            {sentryDsn ? (
```

Comment:
This is not the same as original intend, `sentryDsn` can be "ftp://whatever` and the preference section still renders.

Reply:
Yes, but we set `sentryDsn`. For all intents and purposes it is static. I figured that the purpose of hiding the setting was so that on OpenLens the setting isn't there but on Lens it is there.

Reply:
There are some discussion in original PR related to this, might provide context why isValidDsn is needed
https://github.com/lensapp/lens/pull/3252#issuecomment-875583696

A good example it's our CI in lens-ide is broken, and inject some invalid value in package.json

Reply:
But if you catch the error, like what it done in this PR then you don't have that `uncaught exception` you have the following instead:
![Screen Shot 2021-07-08 at 4 08 44 PM](https://user-images.githubusercontent.com/8225332/124984068-c6859180-e006-11eb-87b0-6b81d670795c.png)

Reply:
Yes, Sentry won't be init, but the UI in preference is still there, and there is no point to show the UI if dsn is not valid.

Please try to change `sentryDsn` in package.json to `http://whatever` and `make dev`
You will still see this in preference
<img width="666" alt="Screenshot 2021-07-08 at 23 20 30" src="https://user-images.githubusercontent.com/1474479/124985454-17b37680-e043-11eb-9347-c7bc2f46579d.png">

Reply:
Ah makes sense, I think it would be better to wait until `Sentry.init()` succeeds.